html-templates/projects/project.tpl

```diff
@@ -141,6 +142,39 @@
                     <li><a class="btn btn-success add-person" href="#add-member" data-toggle="modal">+ {_ "Add"}</a></li>
                 </ul>
             {/if}
+            <hr>
+    <!-- TAGS BLOCK -->
+            {if $Project->Tags}
+                {$Tags = Tag::getTagsWithPrefix($Project->Tags,"tech")}
+                {if $Tags}
+                    {_ "Tech:"}
```

Comment:
@bennovack remove the colon from the translatable string and place it after the `}` is it's more a design element than content, and could be replaced with a different delimeter in the future